src/lib/rest/rest.cpp

```diff
@@ -835,6 +834,14 @@ int servicePathSplit(ConnectionInfo* ciP)
     LM_I(("Service Path %d: '%s'", ix, ciP->servicePathV[ix].c_str()));
   }
 
+  //
+  // stringSplit destroys ciP->servicePath.
```

Comment:
Fixed in acc0506
Using ciP->servicePath[0] wherever possible in calls to metricsMgr.add()